src/common.js

```diff
@@ -67,6 +71,10 @@ var ShellString = function (str, stderr) {
   var that = new String(str);
   that.to = wrap('to', _to, {idx: 1});
   that.toEnd = wrap('toEnd', _toEnd, {idx: 1});
+  that.cat = wrap('cat', _cat, {idx: 1});
```

Comment:
Can this just be `shell.to`? It seems like repeatedly re-wrapping command isn't performant, and it's not DRY.